lambda/variables.tf

```diff
@@ -0,0 +1,255 @@
+# --- lambda/variables.tf ---
+variable "prefix" {
+  description = "A prefix used for all resources"
+  type        = string
+}
+
+variable "project" {
+  description = "A name of the project"
+  type        = string
+}
+
+variable "program" {
+  description = "A name of the program"
+  type        = string
+}
+
+variable "env" {
+  description = "The environment to deploy to"
+  type        = string
+}
+
+variable "region" {
+  description = "The region to deploy to"
+  type        = string
+}
+
+variable "lambda_name" {
+  description = "Lambda function name"
+  type        = string
+}
+
+variable "account_id" {
+  description = "Aws account id"
+  type        = string
+}
+
+variable "timeout" {
+  description = "Amount of time your Lambda Function has to run in seconds"
```

Comment:
Description is inaccurate: this is the duration until which the lambda will be terminated if it hasn't returned.

Reply:
updated